src/SamplesApp/UITests.Shared/Windows_UI_Input/SetProtectedCursor.xaml.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using Uno.UI.Samples.Controls;
+using System.Collections.Generic;
+using System;
+using Windows.Foundation;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Input;
+
+#if HAS_UNO_WINUI
+using Microsoft.UI.Input;
+#endif
+
+namespace SamplesApp.Wasm.Windows_UI_Input
+{
+	[SampleControlInfo("Windows.UI.Input", "SetProtectedCursor", isManualTest: true, description: "Demonstrates use of UIElement.ProtectedCursor / InputSystemCursor / InputSystemCursorShape")]
+	public sealed partial class SetProtectedCursor : Page
+	{
+		public SetProtectedCursor()
+		{
+			this.InitializeComponent();
+		}
+
+		private void ChangeButtonCursor(object sender, PointerRoutedEventArgs e)
+		{
+#if HAS_UNO_WINUI
+			var SizeWestEastCursor = InputSystemCursor.Create(InputSystemCursorShape.SizeWestEast);
+			this.ProtectedCursor = SizeWestEastCursor;
+#endif
```

Comment:
I'm not sure that we WinUI sample app is published?

Reply:
The winui sample app does not exist yet, should be part of the migration to winui by default.